Sources/CvGame.cpp

```diff
@@ -13257,25 +13152,21 @@ void CvGame::findBays()
 void CvGame::findForests()
 {
 	CvPlot* pLoopPlot;
-	int iForestCount;
+	FeatureTypes eForest = (FeatureTypes)GC.getInfoTypeForString("FEATURE_FOREST");
```

Comment:
This should be `const`.

Reply:
done

Reply:
how reliable is the compiler if i add const in the wrong spot?

Reply:
Not certain what do you mean? It will 100% not compile if you try to modify a const object if that is what you mean. However the declaration of const is something you as a coder do to express your intent in code. Even if the compiler didn't care it would still be useful as it gives more information about the variables intention (or functions). 
However in the case of const the compiler cares a **lot**. It will strictly enforce it (unless you are subverting it with `mutable` which should be *very* rare), which helps a lot with enforcing logical consistency. It will also help the optimizer when it comes to deciding what optimization it can perform (you can see an example here: https://www.youtube.com/watch?v=zBkNBP00wJE&t=1635s).